lib/kasket/configuration_mixin.rb

```diff
@@ -79,7 +79,16 @@ def has_kasket_on(*args)
       extend ReadMixin unless methods.map(&:to_sym).include?(:find_by_sql_with_kasket)
     end
 
+    def kasket_expires_in(time)
+      @kasket_expires_in = time
+    end
+
+    def _kasket_expires_in
```

Comment:
it's an internal api, so _kasket_expires_in is not that bad :)

kasket_expires_in_value might work ... or just attr_accessor if you don't like any of these, won't be needed in a bunch of places anyway ...